src/Views/WhatsNewPage.xaml

```diff
@@ -122,10 +127,10 @@
                                     <ResourceDictionary>
                                         <ResourceDictionary.ThemeDictionaries>
                                             <ResourceDictionary x:Key="Light">
-                                                <BitmapImage x:Key="ItemImage" UriSource="{x:Bind LightThemeImage}" />
+                                                <BitmapImage x:Key="ItemImage" UriSource="{x:Bind LightThemeImageBig}" />
                                             </ResourceDictionary>
                                             <ResourceDictionary x:Key="Dark">
-                                                <BitmapImage x:Key="ItemImage" UriSource="{x:Bind DarkThemeImage}" />
+                                                <BitmapImage x:Key="ItemImage" UriSource="{x:Bind DarkThemeImageBig}" />

```

Comment:
Instead of making these properties (I think that's heavier than we want), can you do the resources like Dashboard?
https://github.com/microsoft/devhome/blob/00b2ef18ab159ae000af135b448b1f40a2a53b72/tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml#L25

Reply:
Never mind! Wasn't thinking about future reusability.